### PR TITLE
Added methods to get styles from other data types

### PR DESCRIPTION
**Pull request by blavalla**
Adds methods to get styles from Boolean, Short, Long, Double, Byte, and CharSequence.  Along with the existing methods for Float, Int, and Object, this should cover all of the View methods and fields with ViewDebug annotations so that all properties are shown in the Styles tab of the Elements panel.

Screenshot showing new properties:
screen shot 2016-06-14 at 1 51 53 pm

### CODE REVIEW

File: stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java [hunk 1]
```diff
@@ -298,6 +317,18 @@ private void getStyleFromValue(
       getStyleFromInteger(name, (Integer) value, annotation, styles);
     } else if (value instanceof Float) {
       getStyleFromFloat(name, (Float) value, annotation, styles);
+    } else if (value instanceof Boolean) {
```
Comment (jasta) [inline stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java]: missing Character

File: stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java [hunk 2]
```diff
@@ -243,8 +243,7 @@ private static String mapFlagsToStringUsingAnnotation(
     }
   }
 
-  private static boolean isDefaultValue(
-      Float value) {
+  private static boolean isDefaultValue(Float value) {
```
Comment (jasta) [inline stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java]: Strongly prefer undoing this pattern.  It increases the number of methods for very weak utility.

**Comment by jasta** [comment]
lgtm, will merge after you respond

File: stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java [hunk 3]
```diff
@@ -344,7 +349,63 @@ private void getStyleFromFloat(
...
-    styles.store(name, String.valueOf(value), isDefaultValue(value));
+    styles.store(name, String.valueOf(value), value == 0.0f);
+  }
+
+  private void getStyleFromBoolean(
+      String name,
+      Boolean value,
+      @Nullable ViewDebug.ExportedProperty annotation,
+      StyleAccumulator styles) {
+    styles.store(name, String.valueOf(value), false);
```
Comment (jasta) [inline stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java]: As I look one more time, these methods themselves also do not make sense.  The if / else if blocks above would read so much more succinctly as:

    if (value instanceof Boolean) {
      styles.store(name, String.valueOf(value), false);
    } else { 
      ...

**Comment by jasta** [comment]
One last bit of feedback but it is optional; squash and rebase and we can merge.

**Comment by blavalla** [comment]
I think this should be good to go now.

**Pull request merged**